src/gpio/gpio.c

```diff
@@ -68,7 +68,10 @@ mraa_gpio_close_event_handles_sysfs(int fds[], int num_fds)
     }
 
     for (int i = 0; i < num_fds; ++i) {
-        close(fds[i]);
+        // Check required to avoid closing stdin and of an uninitialized fd
+        if(fds[i] != 0) {
```

Comment:
Haven't noticed this one before, sorry - please add a space between `if` and `(` to align with mraa code style.